File: aio/fashion-statement.py
fashin = open("fashin.txt", "r")
fashout = open("fashout.txt", "w")
 
fare = int(fashin.readline())
nums = [100, 20, 5, 1]
i = 0
counter = 0
 
while fare > 0:
    while fare >= nums[i]:
        fare = fare - nums[i]
        counter += 1
    i += 1
 
print(counter)
 
fashout.write(str(counter))
fashout.close()

File: aio/fashion-statement.cpp
#include <iostream>
#include <cstdio>

int main() {
    int num;
    freopen("fashin.txt", "r", stdin);
    freopen("fashout.txt", "w", stdout);

    std::cin >> num;

    int count = 0;

    while (num >= 100) {
        count ++;
        num = num - 100;
    }

    while (num >= 20) {
        count ++;
        num = num - 20;
    }

    while (num >= 5) {
        count ++;
        num = num - 5;
    }

    while (num >= 1) {
        count ++;
        num = num - 1;
    }

    std::cout << count;

    return 0;
}
